busybox.yaml

```diff
@@ -58,20 +64,17 @@ pipeline:
 
   - name: Install
     runs: |
-      mkdir -p "${{targets.destdir}}"/usr/sbin
       mkdir -p "${{targets.destdir}}"/usr/bin
       mkdir -p "${{targets.destdir}}"/tmp
       mkdir -p "${{targets.destdir}}"/var/cache/misc
-      mkdir -p "${{targets.destdir}}"/bin
-      mkdir -p "${{targets.destdir}}"/sbin
       mkdir -p "${{targets.destdir}}"/etc
       mkdir -p "${{targets.destdir}}"/usr/share/man/man1
       chmod 1777 "${{targets.destdir}}"/tmp
-      install -m755 busybox "${{targets.destdir}}"/bin/busybox
+      install -m755 busybox "${{targets.destdir}}"/usr/bin/busybox
       install -m644 securetty "${{targets.destdir}}"/etc/securetty
 
       mkdir -p "${{targets.destdir}}"/etc/busybox-paths.d
-      ./busybox --list-path > "${{targets.destdir}}"/etc/busybox-paths.d/busybox
+      ./busybox --list-path | sed 's|^bin|usr/bin|;s|^sbin|usr/bin|;s|^usr/sbin|usr/bin|' > "${{targets.destdir}}"/etc/busybox-paths.d/busybox
```

Comment:
Everything into /usr/bin?

Then our links can be just 'ln -s busybox other-thing'

Reply:
Yes, but after this lands; dust settles; and apko is changed to produce usr-merged layout by default.

Because freshly built images for now will continue to use /bin/busybox for the symlink target.

Reply:
We could instead of writing our own implementation ship suse's bash converter - https://build.opensuse.org/projects/openSUSE:Factory/packages/compat-usrmerge/files/convertfs

or debian's perl converter - https://salsa.debian.org/md/usrmerge/-/blob/master/convert-usrmerge?ref_type=heads

Reply:
I just want to point a tool at a tree (${{targets.contextdir}}) and say "merge the usr here!".
it should be able to always pass.  but if it fails it should fail the build.

I'm sure I'm missing something and it is more complicated than i'm imagining.

then we run that from a pipeline as the last step of a package (and each subpackage).

i really want to be able to tell melange "always run these pipelines".